src/App.js

```diff
@@ -18,7 +16,8 @@ import NotFound from "./components/NotFound";
 import { reducer } from "./utils/reducer";
 import { StateContext } from "./utils/stateContext";
 import SignUpForm from "./components/SignUpForm";
-import { getWineListings } from './services/wineServices';
+import { getWineListings } from "./services/wineServices";
```

Comment:
This line of code is a duplication of what is already in main. We can let is merge to main again and see what happens?

Reply:
I think this is from deleting the line and re-adding it. It's still in the file